dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/command/CancelQueueCommand.java

```diff
@@ -23,14 +23,19 @@
 import io.netty.handler.codec.http2.Http2Error;
 
 public class CancelQueueCommand extends QueuedCommand {
+    private final Http2Error error;
```

Comment:
Why add error?

Reply:
`Reset` frame requires a `errorcode` , `CancelQueueCommand` is used to send reset frame to remote peer not only `cancel`  .

Reply:
But now it seems to execute cancel after rst received?

Reply:
Yep , `rst` acts as `Cancel` at app layer , but at the sender side ,  `rst` may mean `Close the stream now` not only cancel